### PR TITLE
Remove unused code related to old Stripe settings

### LINKED ISSUE #2431: Remove remaining traces (`generate_stripe_account_keys_html( ... )`) of old settings
**Describe the bug**
This is not exactly a bug. Instead, it consists of a function (`generate_stripe_account_keys_html( ... )`) that exists in two places but they are not being used anymore and should be removed.

**To Reproduce**
1. View [class-wc-gateway-stripe.php#L1047](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/72578ec87bc59f64276985972142cd3fbfe69a4d/includes/class-wc-gateway-stripe.php#L1047-L1082).
2. View [class-wc-stripe-upe-payment-gateway.php#L1217](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/72578ec87bc59f64276985972142cd3fbfe69a4d/includes/payment-methods/class-wc-stripe-upe-payment-gateway.php#L1217-L1252).

**Expected behavior**
The method `generate_stripe_account_keys_html( ... )` should be removed from both `class-wc-gateway-stripe.php` and `class-wc-stripe-upe-payment-gateway.php`.

**Environment (please complete the following information):**
 - Stripe Plugin Version: 6.7.0

**Additional context**
* #2166

### PR DESCRIPTION
**Pull request by asumaran**
Fixes #2431

## Changes proposed in this Pull Request:

Removes unused code that should've been removed in #2166 from the following files:

- `includes/class-wc-gateway-stripe.php`
- `includes/connect/class-wc-stripe-connect.php`
- `includes/payment-methods/class-wc-stripe-upe-payment-gateway.php`

Also, removes the minified file `assets/js/stripe-admin.min.js` given that the source file `assets/js/stripe-admin.js` was already removed in #2166

## Testing instructions

There's no way to test this. The removed code are some leftovers from the [original PR](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2166) where we wanted to remove the deprecated Stripe settings.

Just in case, we should try setting up the Stripe keys the usual way, to make sure it works.

- Go to WooCommerce > Settings > Payments
- Click "Manage" on the "Stripe" row.
- Click the "Settings" tab.
- Click "Edit account keys" in the "General" section.
- On the "Test" tab enter the "Test publishable key" and the "Test secret key"
- Click "Test connection". Connection should be successfull.
- Click "Save test keys". Keys should be persisted without issues.

---

-   [ ] Covered with tests (or have a good reason not to test in description ☝️)
-   [x] Added changelog entry **in both** `changelog.txt` and `readme.txt` (or does not apply)
-   [ ] Tested on mobile (or does not apply)

**Post merge**

-   [ ] Added testing instructions to the [Release Testing Instructions wiki page](https://github.com/woocommerce/woocommerce-gateway-stripe/wiki/Release-Testing-Instructions) (or does not apply)

### CODE REVIEW

File: includes/connect/class-wc-stripe-connect.php [hunk 1]
```diff
@@ -90,27 +90,7 @@ public function maybe_handle_redirect() {
 				wp_safe_redirect( esc_url_raw( remove_query_arg( [ 'wcs_stripe_state', 'wcs_stripe_code' ] ) ) );
 				exit;
```
Comment (lovo-h) [inline includes/connect/class-wc-stripe-connect.php]: Nit: Could we remove this empty line?

File: assets/js/stripe-admin.min.js [hunk 2]
```diff
@@ -1 +0,0 @@
...ss("disabled"),n("#wc_stripe_connect_button").on("click",function(){return!1}),n("#woocommerce_stripe_api_credentials").next("p").append(" (Please save changes before selecting this button.)"),n("form").find("input, select").off("change input",e)}),n(".wc_gateways").on("click",".wc-payment-upe-method-toggle-enabled, .wc-payment-upe-method-toggle-disabled",function(){var e=n(this).find(".woocommerce-input-toggle");return e.toggleClass("woocommerce-input-toggle--enabled  woocommerce-input-toggle--disabled"),e.parent().toggleClass("wc-payment-upe-method-toggle-enabled  wc-payment-upe-method-toggle-disabled"),n("#wc_stripe_upe_change_notice").removeClass("hidden"),!1}),n("#mainform").submit(function(){var e=n(this);n(".wc_gateways .wc-payment-upe-method-toggle-enabled").each(function(){e.append('<input type="hidden" name="woocommerce_stripe_upe_checkout_experience_accepted_payments[]" value="'+n(this).closest("tr").data("upe_method_id")+'" />')})});var e=Date.now()/1e3-wc_stripe_settings_params.time;240<Math.abs(e)&&n("#woocommerce_stripe_test_webhook_secret, #woocommerce_stripe_webhook_secret").closest("td").append("<p>"+wc_stripe_settings_params.i18n_out_of_sync+"</p>")}};e.init()});
```
Comment (lovo-h) [inline assets/js/stripe-admin.min.js:1]: Good catch 👍

File: includes/connect/class-wc-stripe-connect.php (hunk 1 above)
Comment (asumaran) [reply includes/connect/class-wc-stripe-connect.php]: done [beefaa3](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2432/commits/beefaa31ebb5180127e882889800f65246bb4aae)

**Review by lovo-h — APPROVED** [review]
Code changes look good and test well.

**Tests**
* ✅ Keys should be persisted without issues.

**Pull request merged**